admin-dev/themes/default/template/helpers/list/list_header.tpl

```diff
@@ -294,17 +294,22 @@
 								{$params.title}
 							{/if}
 							{if (!isset($params.orderby) || $params.orderby) && !$simple_header && $show_filters}
+                {if $frameworkIndexUrl}
+                    {assign var="baseSortUrl" value="$frameworkIndexUrl"}
+                {else}
+                    {assign var="baseSortUrl" value="{$currentIndex}&token={$token}"}
```

Comment:
I think the problem you fixed with the last commit was because there was a double escape, in the variable and then in the href again. Now it seems better for the currentIndex case but you still perform additional escape for the framework one so I would suggest here:

```suggestion
                    {assign var="baseSortUrl" value="{$currentIndex|escape:'html':'UTF-8'}&token={$token|escape:'html':'UTF-8'}"}
```

Like you did before, and remove it below (see other comment)